public/form/css/styles.css

```diff
@@ -705,23 +716,11 @@ hr {
 }
 
 .bg.log {
-  background-image: url('../img/sky.png');
```

Comment:
I talk with Daniel about that change and he say me that is no necessary have two version of the same image in that case. @tiagojsag @clementprdhomme